File: All_Vibe_Coder_Skills_v2 with master_instruction/all_vibe_coders/nodejs-backend-vibe-coder/src/routes/items.js
import { Router } from 'express';

export const itemsRouter = Router();

itemsRouter.get('/', async (_req, res, next) => {
  try {
    // Replace with real service call
    res.json([{ id: 1, name: 'Hello Copilot! Follow SKILL.md rules.' }]);
  } catch (err) {
    next(err);
  }
});

File: All_Vibe_Coder_Skills_v2 with master_instruction/all_vibe_coders/fastapi-vibe-coder/app/routers/items.py
from fastapi import APIRouter
from pydantic import BaseModel

router = APIRouter(prefix="/items", tags=["items"])


class ItemResponse(BaseModel):
    id: int
    name: str


@router.get("/", response_model=list[ItemResponse])
async def list_items():
    # Replace with real DB query
    return [{"id": 1, "name": "Hello Copilot! Follow SKILL.md rules."}]
